kernel/Kconfig

```diff
@@ -9,6 +9,18 @@ module = KERNEL
 module-str = kernel
 source "subsys/logging/Kconfig.template.log_config"
 
+config KERNEL_WARN_ALLOC_INTERVAL
+	int "Interval of pending wait for event warnings"
+	default 1
```

Comment:
If this is something we want, it should be an opt-in feature instead of opt-out, as this feels like a debugging tool.